README.md

```diff
@@ -136,24 +136,68 @@ inbound or outbound value is nil.
 If a custom type is registered for a record-type field, then any `to_avro` 
 method/Proc should return a Hash with string keys for encoding using Avro.
 
-#### Encode/Decode
+### Encoding and Decoding
 
-Models can be encoded using Avro leveraging a schema registry to encode a schema
-id at the beginning of the value.
+`Avromatic` provides two different interfaces for encoding the key (optional)
+and value associated with a model.
+
+#### Manually Managed Schemas
+
+The attributes for the value schema used to define a model can be encoded using:
+
+```ruby
+encoded_value = model.avro_encoded_value
```

Comment:
Maybe we can brainstorm naming alternatives offline but I'm a bit worried that developers will confuse `avro_encoded_value` / `avro_message_value` and `decode` / `deserialize`.